### PR TITLE
Update to Batch data plane 6.0.0 and Storage 8.0.1

### PR DESCRIPTION
**Pull request by matthchr**

### CODE REVIEW

**Comment by matthchr** [comment]
@tamram Take a look at the changes under the ArticleSamples folder: I ran all of the article sample projects (and they worked) except these:
- DotNetTutorial
- PersistOutputs - Needs a manually uploaded application package which I didn't do
- MultiInstanceTask - Needs a manually uploaded application package which I didn't do

@itowlson @xingwu1 @darylmsft - I ran all of the other samples and they worked.

Thanks,
-Matt

**Comment by matthchr** [comment]
@tamram @itowlson @xingwu1 -- pinging on this

File: CSharp/BatchMetrics/BatchMetrics/packages.config [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+  <package id="Microsoft.Data.Edm" version="5.7.0" targetFramework="net452" />
+  <package id="Microsoft.Data.OData" version="5.7.0" targetFramework="net452" />
+  <package id="Microsoft.Data.Services.Client" version="5.7.0" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.5" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime.Azure" version="3.3.5" targetFramework="net452" />
   <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net452" />
-  <package id="System.Spatial" version="5.6.4" targetFramework="net452" />
-  <package id="WindowsAzure.Storage" version="6.2.0" targetFramework="net452" />
+  <package id="System.Spatial" version="5.7.0" targetFramework="net452" />
+  <package id="WindowsAzure.Storage" version="8.0.1" targetFramework="net452" />
```
Comment (itowlson) [inline CSharp/BatchMetrics/BatchMetrics/packages.config]: Do we still need Storage and its dependencies in the BatchMetrics project?

File: CSharp/BatchMetrics/BatchMetricsUsageSample/packages.config [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+  <package id="Microsoft.Data.Edm" version="5.7.0" targetFramework="net452" />
+  <package id="Microsoft.Data.OData" version="5.7.0" targetFramework="net452" />
+  <package id="Microsoft.Data.Services.Client" version="5.7.0" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.5" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime.Azure" version="3.3.5" targetFramework="net452" />
   <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net452" />
-  <package id="System.Spatial" version="5.6.4" targetFramework="net452" />
-  <package id="WindowsAzure.Storage" version="6.2.0" targetFramework="net452" />
+  <package id="System.Spatial" version="5.7.0" targetFramework="net452" />
+  <package id="WindowsAzure.Storage" version="8.0.1" targetFramework="net452" />
```
Comment (itowlson) [inline CSharp/BatchMetrics/BatchMetricsUsageSample/packages.config]: Again, not sure we need Storage here, or is it that the common project brings it in?

Comment (matthchr) [inline CSharp/BatchMetrics/BatchMetricsUsageSample/packages.config]: Good eye -- removed.

File: CSharp/BatchMetrics/BatchMetrics/packages.config (hunk 1 above)
Comment (matthchr) [inline CSharp/BatchMetrics/BatchMetrics/packages.config]: Good eye -- removed.

**Pull request merged**